prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -1538,8 +1538,9 @@ int efa_rdm_ope_post_remote_write(struct efa_rdm_ope *ope)
 		if (ope->fi_flags & FI_INJECT) {
 			assert(ope->iov_count == 1);
 			assert(ope->total_len <= ep->inject_size);
-			copied = ofi_copy_from_hmem_iov(pkt_entry->wiredata + sizeof(struct efa_rdm_rma_context_pkt),
-				ope->total_len, FI_HMEM_SYSTEM, 0, ope->iov, ope->iov_count, 0);
+			OFI_DBG_SET(copied,
```

Comment:
I am not that great with macros, what is this doing?
```
#ifndef NDEBUG
#define OFI_DBG_VAR(type, name) type name;
#define OFI_DBG_SET(name, val) name = val
#define OFI_DBG_ADD(name, val) name += val
#define OFI_DBG_CALL(func) func
#else
#define OFI_DBG_VAR(type, name)
#define OFI_DBG_SET(name, val) do {} while (0)
#define OFI_DBG_ADD(name, val) do {} while (0)
#define OFI_DBG_CALL(func) do {} while (0)
#endif
```

Reply:
actually, there is a bug here. Let me updtae the patch.

Reply:
@a-szegel the macro is a noop when NDEBUG is not set

Reply:
We can't use the macro here since the action needs to be done.

Reply:
thats what I though but wasn't sure.